stages/test/test_cloud-init.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+
+import os.path
+
+from osbuild.testutil.imports import import_module_from_path
+
+
+# Test that the 'datasource_list' option is dumped as a single line
+# in the configuration file.
+def test_cloud_init_datasource_list_single_line(tmp_path):
+    treedir = os.path.join(tmp_path, "tree")
```

Comment:
(nitpick) the new python `pathlib` allows this to be written as `treedir = tmp_path / "tree"` which is slightly shorter (and after some getting used to I find it actually quite nice to read).

Reply:
Thanks, I didn't know about this, will have a look... :thinking:

Reply:
done